index.html

```diff
@@ -38,8 +38,8 @@ <h2>Open source re-implementation of RollerCoaster Tycoon 2</h2>
         <div class="information">How to play OpenRCT2</div>
       </li>
       <li>
-        <a href="blog/2020/survey2020.html">2020 User Survey Results</a>
```

Comment:
Is this going dodo? Are there other links to that page?

Reply:
I don’t think we have a page that lists all of our blogs. But keeping it on the front page when it was so outdated didn’t seem like a good idea either.

Reply:
Maybe adding an "Archive" button, and that being just a list of old posts?